quickwit-metastore/src/metastore/file_backed_metastore/index.rs

```diff
@@ -205,10 +206,21 @@ impl Index {
             });
         }
 
-        self.index.update_timestamp = now_timestamp;
         Ok(())
     }
 
+    /// Helper to publish a list of splits.
+    pub(crate) fn publish_splits<'a>(
```

Comment:
Can we call the helper function `publish_splits_helper` or `publish_splits_inner`? When I quickly glanced over the file initially, I couldn't tell which function was calling the other. That might be because I'm reviewing a diff and not the full file.

Reply:
Rename as _helper and added a comment